libass/ass.c

```diff
@@ -893,9 +897,13 @@ static int process_fonts_line(ASS_Track *track, char *str)
             decode_font(track);
         }
         track->parser_priv->fontname = strdup(p);
-        ass_msg(track->library, MSGL_V, "Fontname: %s",
-               track->parser_priv->fontname);
-        return 0;
+        if (track->parser_priv->fontname) {
```

Comment:
I think it's better to use here
```c
if (!track->parser_priv->fontname)
    return -1;
```
for consistency and clarity.

Reply:
Done.